src/services/geocoding.ts

```diff
@@ -261,7 +268,7 @@ async function reverse(
   position: Position,
   options: ReverseGeocodingOptions = {}
 ): Promise<GeocodingSearchResult> {
-  if (!Array.isArray(position) || position.length < 2) {
+  if (position.length !== 2) {
```

Comment:
Please keep the Array validation. This is mainly intended for non-TS as coordinates are also ofter given as object `{lon: number, lat: number}`